code/__DEFINES/dcs/signals.dm

```diff
@@ -872,6 +872,10 @@
 ///from monkey CtrlClickOn(): (/mob)
 #define COMSIG_XENO_MONKEY_CLICK_CTRL "xeno_monkey_click_ctrl"
 
+// /datum/component/atmos_sensitive
```

Comment:
its an element, fix the comments pls (I was looking for a component and couldnt find it)

Reply:
actually why does it mention the element anyways? The signal is not linked to the element in any way and comments shouldn't give the idea that they're made for specific things; they just represent the action of temp exposure happening.

Reply:
actually why does it mention the element anyways? The signal is not linked to the element in any way and comments shouldn't give the idea that they're made for specific things; they just represent the action of temp exposure happening.

Reply:
local man refactors his heat sensitivity system 3 times, forgets how it works when writing documentation

Reply:
its ok old man ur almost there